### PR TITLE
Take Neo4J out of tunnel name

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by scott-ace-newton — APPROVED** [review]

File: tunnel-registrator.service [hunk 1]
```diff
@@ -13,7 +13,7 @@ ExecStart=/bin/bash -c '\
   KONSTRUCTOR_API_KEY=$(/usr/bin/etcdctl get /ft/_credentials/konstructor/api-key); \
   PRIVATE_IP=$(curl -s 169.254.169.254/latest/meta-data/local-ipv4); \
   docker run --rm --name %p_$(uuidgen) \
-  -e="DOMAINS=$ENV-neo4j-tunnel-up" \
+  -e="DOMAINS=$ENV-tunnel-up" \
```
Comment (kghatton) [inline tunnel-registrator.service:16]: Isn't this going to evaluate to the same name as the corresponding delivery cluster - whose address you probably don't want to overwrite?

Comment (ghost) [reply tunnel-registrator.service:16]: We're changing the environment tag to not be the same as the delivery cluster.  The new tag will be *-data (e.g. pre-prod-uk-data), so it wouldn't overwrite.

Comment (kghatton) [reply tunnel-registrator.service:16]: You could put `-data` here to prevent anyone making that mistake.

**Comment by ghost** [comment]
We're not worried about this at the moment, the name being obvious is probably better for now.

**Pull request closed without merging**